postgresql/deploy.nix

```
@@ -0,0 +1,223 @@
+let
+  region = "eu-central-1";
+  accessKeyId = "supabase-dev"; ## aws profile
+  sysbench = pkgs.callPackage ./sysbench.nix { postgresql = pkgs.postgresql_14; };
```

[overrideAttrs](https://nixos.org/manual/nixpkgs/stable/#sec-pkg-overrideAttrs):

```suggestion
  sysbench = pkgs.sysbench.overrideAttrs (oldAttrs: rec {
      configureFlags = oldAttrs.configureFlags ++  [ "--with-pgsql" "--without-mysql" ];
    });
```